frontend_app_file/test/apiMock.js

```diff
@@ -0,0 +1,44 @@
+const nock = require('nock')
+
+const mockPutMyselfFileRead200 = (apiUrl, workspaceId, contentId) => {
+  return nock(apiUrl)
+    .persist()
```

Comment:
It would be better to avoid having to put .persist() in every mock.

try check the documentation to find a better way of mocking api using nock or other things

I you find that it's a not an issue to use .persist(), I'm fine with it

Reply:
While the documentation of Nock don't mention any downside of using persist, I think we could use it but I removed it in `apiMock.js` in order to use it only when necessary in the actual test file, like `mockPutMyselfFileRead200.persist()` 

Note : We could also add a `nock.cleanAll()` or `scope.persist(false)` after the component shallow, but the `shallow` rendering is asynchronous so it will clear the nock when not all api call where sent or served.